CHANGELOG.md

```diff
@@ -7,21 +7,15 @@
     Click to see more.
   </summary>
 
-### New Features
-
-- Loading of default dashboards on bootstrap ([#71](https://github.com/src-d/superset-compose/issues/71)).
-- Update gitbase to v0.20.0 ([#81](https://github.com/src-d/superset-compose/issues/81)).
-- Update bblfsh to v2.14.0 ([#81](https://github.com/src-d/superset-compose/issues/81)).
-- Use sparksql instead of mysql for enterprise version ([#79](https://github.com/src-d/superset-compose/issues/79)).
-- Cancel query to database on stop ([#35](https://github.com/src-d/superset-compose/issues/35)).
-
 </details>
 
-## v0.0.1 - 2019-05-16
+## [v0.1.0](https://github.com/src-d/sourced-ui/releases/tag/v0.1.0) - 2019-06-03
 
-The `srcd/superset` docker image is based on Superset 0.32, and contains the following additions:
-- an extra tab, UAST, to explore bblfsh parsing results.
```

Comment:
should we mention creation of default user?

Reply:
:+1: added. Already squashed since it's a small change.